.github/ISSUE_TEMPLATE.md

```diff
@@ -1 +1,26 @@
 **Important**: This is a public repository. Anyone in the world can see what's posted here. If you are posting screenshots or log files, please **carefully examine them for** the presence of any kind of **protected health information** (PHI). Images or logs containing PHI _must_ be posted in fully-redacted form, with no visible PHI.
+
+---
+
+If you're reporting a bug copy the following template below, replacing the _italicised_ text with your own. This is important to ensure we can look at your bug quickly and efficiently, and will improve the speed in which it gets fixed :-)
+
+---
+
+_Brief description of bug encountered_.
+
+**Steps to reproduce**:
+ - _Write individual steps here_
+ - _That can be used to reproduce_
+ - _The problem_
+
+**What should happen**:
+ - _Detail what you **expect** to happen when you complete the last step_
+
+**What actually happens**:
+ - _Describe what happens (the bug!)_
```

Comment:
If possible include a link to a doc/page that exhibits this bug - this would save me so much time!

Reply:
Also, screenshots!